cmd/run.go

```diff
@@ -461,14 +466,23 @@ func RunServer(ctx context.Context, config *Config) error {
 		return fmt.Errorf("failed to initialize authenticator: %w", err)
 	}
 
+	ctxtagsOpts := []grpc_ctxtags.Option{
+		grpc_ctxtags.WithFieldExtractor(grpc_ctxtags.TagBasedRequestFieldExtractor("store_id")),
```

Comment:
After looking at this in more detail, I don't think this package is going to work for us. For starters, it only allows extraction of one field from what I can tell. Also, it doesn't work at all even just for one tag 😄 . It seems it depends on custom struct tags generated from a different protoc plugin, and from the examples in the repo I cannot tell which it is.

I may suggest we just create a `internal/middleware/storeid` and `internal/middleware/modelid` package that extracts the StoreID and AuthorizationModelID from the request manually and injects it into the context, and then this logging package can extract it from the context (if it exists) and add it to the logged fields.

For example,
```
// internal/middleware/storeid.go
package storeid

type ctxKey string

const (
    storeIdContextKey = "store-id"
)

func ContextWithStoreId(parent context.Context, storeID string) context.Context {
    ...
}

func StoreIdFromContext(ctx context.Context) (string, bool) {
    ...
}

type requestWithStoreID interface {
    GetStoreId()
}

func NewStoreIdUnaryInterceptor() grpc.UnaryServerInterceptor {
    return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler rpc.UnaryHandler) (interface{}, error) {
        r, ok := req.(requestWithStoreId)
        if ok {
            ctx = ContextWithStoreId(ctx, r.GetStoreId())
        }
}
}
```

Reply:
That would be nice actually. Then we don't even need to pass it in the methods.